src/Organizations/OrganizationForm/OrganizationSummaryForm/FieldOrganizationTypes/FieldOrganizationTypes.js

```diff
@@ -20,7 +20,7 @@ export const FieldOrganizationTypes = () => {
 
   const itemToString = item => item;
 
-  const typeOptions = organizationTypes?.map(({ id }) => id);
+  const typeOptions = organizationTypes?.filter(type => type.status === 'Active').map(({ id }) => id);
```

Comment:
it would be better to keep statutes as const

Reply:
I believe, @AndreiShumski2 meant to use `const` for type statuses like `const TYPE_STATUSES = { active: 'Active", inactive: 'Inactive' ... }` and use it in code

Reply:
smth like we have for [Organization](https://github.com/folio-org/ui-organizations/blob/master/src/common/constants/organization.js#L4)

Reply:
Thanks! I added ORGANIZATION_TYPES_STATUS